qa/acceptance-tests/src/test/java/io/camunda/it/rdbms/db/usagemetrics/UsageMetricsIT.java

```diff
@@ -61,110 +77,176 @@ private void writeMetric(
+    final var actual = usageMetricReader.usageMetricStatistics(filter);
+    final var actualTU = usageMetricTUReader.usageMetricTUStatistics(filter);
 
     // then
     assertThat(actual)
         .isEqualTo(
             new UsageMetricStatisticsEntity(
-                16,
-                14,
+                19,
+                17,
                 2,
                 Map.of(
-                    "tenant1",
+                    TENANT1,
                     new UsageMetricStatisticsEntityTenant(13L, 11L),
-                    "tenant2",
-                    new UsageMetricStatisticsEntityTenant(3L, 3L))));
+                    TENANT2,
+                    new UsageMetricStatisticsEntityTenant(6L, 6L))));
+
+    assertThat(actualTU)
+        .isEqualTo(
+            new UsageMetricTUStatisticsEntity(
+                5,
+                Map.of(
+                    TENANT1,
+                    new UsageMetricTUStatisticsEntityTenant(2L),
+                    TENANT2,
+                    new UsageMetricTUStatisticsEntityTenant(3L))));
   }
 
   @TestTemplate
   public void shouldAggregateMetricsWithoutTenant() {
     // given
-    final var now = OffsetDateTime.now();
-    writeMetric(usageMetricWriter, RPI, now, TENANT1, 11L);
-    writeMetric(usageMetricWriter, EDI, now, TENANT1, 11L);
-    writeMetric(usageMetricWriter, RPI, now.minusMinutes(5), TENANT1, 2L);
-    writeMetric(usageMetricWriter, RPI, now.minusMinutes(5), TENANT2, 3L);
-    writeMetric(usageMetricWriter, EDI, now.minusMinutes(5), TENANT2, 3L);
+    writeMetric(usageMetricWriter, RPI, NOW, TENANT1, 11L);
+    writeMetric(usageMetricWriter, EDI, NOW, TENANT1, 11L);
+    writeMetric(usageMetricWriter, ATU, NOW, TENANT1, 2L);
+    writeTUMetric(usageMetricTUWriter, NOW, TENANT1, ASSIGNEE_HASH_1);
+    writeTUMetric(usageMetricTUWriter, NOW, TENANT1, ASSIGNEE_HASH_2);
+    writeMetric(usageMetricWriter, RPI, NOW_MINUS_5M, TENANT1, 2L);
+    writeMetric(usageMetricWriter, RPI, NOW_MINUS_5M, TENANT2, 3L);
+    writeMetric(usageMetricWriter, EDI, NOW_MINUS_5M, TENANT2, 3L);
+    writeMetric(usageMetricWriter, ATU, NOW_MINUS_5M, TENANT2, 3L);
+    writeTUMetric(usageMetricTUWriter, NOW_MINUS_5M, TENANT2, ASSIGNEE_HASH_1);
+    writeTUMetric(usageMetricTUWriter, NOW_MINUS_5M, TENANT2, ASSIGNEE_HASH_2);
+    writeTUMetric(usageMetricTUWriter, NOW_MINUS_5M, TENANT2, ASSIGNEE_HASH_3);
     rdbmsWriter.flush();
 
     // when
-    final var actual =
-        usageMetricReader.usageMetricStatistics(new UsageMetricsFilter.Builder().build());
+    final UsageMetricsFilter filter = new Builder().build();
+    final var actual = usageMetricReader.usageMetricStatistics(filter);
+
+    final var actualTU = usageMetricTUReader.usageMetricTUStatistics(filter);
 
     // then
     assertThat(actual).isEqualTo(new UsageMetricStatisticsEntity(16, 14, 2, null));
+    assertThat(actualTU).isEqualTo(new UsageMetricTUStatisticsEntity(5, null));
```

Comment:
❓ Shouldn't this be 3 instead of 5? There is only 3 unique task users according to the hashes, and the test mentions it's supposed to be without tenant. So I assume the tenant is not relevant here.

Reply:
I agree, it should be 3.